tests-integration/go.mod

```diff
@@ -3,12 +3,12 @@ module tests
 go 1.18
 
 require (
-	github.com/influxdata/influxdb-observability/common v0.2.28
+	github.com/influxdata/influxdb-observability/common v0.2.30
 	github.com/influxdata/line-protocol/v2 v2.2.1
-	github.com/influxdata/telegraf v0.0.0-20221005200317-6606c5ed33ac
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/influxdbexporter v0.61.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/extension/healthcheckextension v0.61.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/influxdbreceiver v0.61.0
+	github.com/influxdata/telegraf v0.0.0-20221027194347-99a48f460e67
```

Comment:
after the release next week should this get changed to v1.24.3?

Reply:
It wouldn't hurt, but it wouldn't help either.